pkg/server/registry/apigroups/acorn/projects/storage.go

```diff
@@ -98,7 +99,10 @@ func migrateLegacyNamespaces(ctx context.Context, c kclient.Client) error {
 	}).Add(*notManaged)
 	namespaces := corev1.NamespaceList{}
 	if err := c.List(ctx, &namespaces, kclient.MatchingLabelsSelector{Selector: nsSelector}); err != nil {
-		return err
+		// If the namespace list call fails, then we just ignore it and move on. This is necessary to
+		// ensure that the migration doesn't block Manager start-up where namespaces don't exist.
+		logrus.Debugf("namespace list call failed during migration of legacy namespaces, ignoring and moving on: %v", err)
```

Comment:
nit: `logrus.WithError(err).Debug("namespace list call failed during migration of legacy namespaces, ignoring and moving on")